securedrop/create-dev-data.py

```diff
@@ -56,8 +49,13 @@ def main(staging=False):
                                                 first_name="Clark",
                                                 last_name="Kent")
 
+        NUM_SOURCES = os.getenv('NUM_SOURCES', 2)
+        if NUM_SOURCES == "ALL":
+            # We want all strings, 14 sources will give all the strings based
+            # on current string count.
+            NUM_SOURCES = 14
```

Comment:
Technically this is 1 source too many since there are 25 strings and 2 strings per source, so we consume 28 strings, meaning one fully repeated couple.

Reply:
elegantly resolved by @creviera in c83d8c4ab3c41c8cdd1362f220b99bdc7489a770